fcrepo-api-x-jena/src/main/java/org/fcrepo/apix/jena/impl/LookupOntologyRegistry.java

```diff
@@ -117,24 +121,18 @@ public void delete(final URI uri) {
         registry.delete(uri);
     }
 
-    /** Try infinitely to read contents of registry in order to index ontologyIRIs */
+    /** Load ontologies from underlying registry, if present. */
     public void init() {
+        if (ontologyIRIsToLocation != null) {
```

Comment:
This was (as the commit says) hack, but one that is not really impactful in practice.  The hack is to allow the service to initialize (and be available in an OSGi sense) before the underlying container exists, in cases where the service is not configured to create its own container.  Proper behaviour would be to fail all requests until the underlying registry is accessible, or fail to initialize in the first place (which is what the previous code did).   This hack is temporary and will be resolved as part of the process to fix the ITs (which were redacted from this PR), but was necessary to get this PR out so that people can look at.